app/src/main/java/com/x8bit/bitwarden/ui/tools/feature/generator/util/PolicyInformationPasswordGeneratorExtensions.kt

```diff
@@ -10,7 +11,14 @@ fun List<PolicyInformation.PasswordGenerator>.toStrictestPolicy():
     PolicyInformation.PasswordGenerator {
     return PolicyInformation.PasswordGenerator(
         capitalize = mapNotNull { it.capitalize }.any { it },
-        defaultType = firstNotNullOfOrNull { it.defaultType },
+        overridePasswordType = mapNotNull { it.overridePasswordType }.reduceOrNull { acc, value ->
+            when {
+                // password should be prioritized over passphrase, else we keep the value
+                acc == TYPE_PASSWORD -> acc
+                value == TYPE_PASSWORD -> value
+                else -> value
```

Comment:
So if I understand this, you are saying that if the list contains a password type, then that should be the override type but if it doesn't contain a password type then we want to use the last type in the list?

Reply:
This can only be password or passphrase, (as mapNotNull is excluding the nulls).

So if it is not password, will always be passphrase.

Seemed redundant to add a `value == TYPE_PASSPHRASE -> value`, but if that adds clarity, might be worthy to add